app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    @posts = Post.all.order(created_at: :desc).page(params[:page]).per(5)
+  end
```

Comment:
Love the idea of creating a separate homepage controller to display the posts, great SRP